Sources/Public/Configuration/LottieConfiguration.swift

```diff
@@ -44,4 +46,11 @@ public struct LottieConfiguration: Hashable {
   /// The color space to be used for rendering
   ///  - Defaults to `CGColorSpaceCreateDeviceRGB()`
   public var colorSpace: CGColorSpace
+
+  /// Whether or not transform and position changes of the view should animate alongside
+  /// any existing animation context.
+  ///  - Defaults to `true` which will grab the current animation context and animate position and
+  ///    transform changes matching the current context's curve and duration.
+  ///    `false` will cause transform and position changes to happen unanimated
+  public var animatesLayoutWithinCurrentContext: Bool
```

Comment:
What do you think about a name like this instead? I like the idea of using the word "inherit", and explicitly mentioning that this is coming from the set of Core Animation animations.

```suggestion
  public var layoutChangesInheritCurrentCoreAnimationContext: Bool
```

Reply:
`animateLayoutChangesWithCurrentCoreAnimationContext` makes sense too. "Current Context" by itself seems a bit ambiguous.